### ISSUE TITLE
Permanent subscriptions (it was "[BUG] in POST v2/subscriptions with Permanent subscription")

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#1949]
in POST v2/subscriptions with Permanent subscription.
> expires: Subscription expiration date in ISO8601 format. Permanent subscriptions must omit this field.

#### Subcription request
```
POST http://localhost:1026/v2/subscriptions
Content-Type: application/json
Fiware-Service: test_casub_happy_path
Fiware-ServicePath: /test
 {
 	"description": "my first subscription",
 	"subject": {
 		"entities": [{
 			"idPattern": ".*",
 			"type": "room"
 		}],
 		"condition": {
 			"attributes": ["temperature_0", "temperature_1", "temperature_2"],
 			"expression": {
 				"q": "temperature>40"
 			}
 		},
 		"notification": {
 			"callback": "http://localhost:1234",
 			"throttling": 5,
 			"attributes": ["temperature_0", "temperature_1", "temperature_2"]
 		}
 	}
 }
```

#### subscription response
```
http code: 400
date: Thu, 24 Mar 2016 10:21:14 GMT
connection: Keep-Alive
content-type: application/json
content-length: 79
{"error":"BadRequest","description":"no expiration for subscription specified"}
```

#### Expected response
```
http code: 201
```

**Comment by fgalan** [comment]
This is not actually a bug, but functionality not developed. In fact, probably we have a "backlog" issue about supporting permanent subscriptions (unfortunatelly, I haven't found it). However, you can keep this a a bug labeled issue if that eases the task of keep tracking of this for you.

**Comment by fgalan** [comment]
Re-converte in a backlog issue.

**Comment by fgalan** [comment]
Implemented in PR #2033. Moved to QATestPending milestone to wait for the functional test implementation.

**Comment by iariasleon** [comment]
LGTM
```
http code: 201 - Created
headers:
   date: Fri, 17 Jun 2016 10:21:01 GMT
   fiware-correlator: 30f67086-3475-11e6-99ed-005056a20feb
   connection: Keep-Alive
   content-length: 0
   location: /v2/subscriptions/5763cf0d3c1e802b29bbc562
```
Also in DB is stored (Wed, 14 Mar 2255 16:00:00 GMT): 
```
"expiration": 9000000000000000000L
```
